packages/js/src/ui/components/Header/StatusOptions.tsx

```diff
@@ -20,7 +19,7 @@ export const StatusOptions = (props: { setFeedOptions: (options: FetchFeedArgs)
          * TODO: Implement setFeedOptions and isSelected after Filter is implemented
          */
         onClick={() => {
-          props.setFeedOptions({ status: NotificationStatus.UNREAD });
+          props.setFeedOptions({ read: true });
```

Comment:
not sure what will be an appropriate status for fetching both unread and read notifications